scraper/src/config/config_loader.py

```diff
@@ -74,7 +78,7 @@ def __init__(self, config):
         data = self._load_config(config)
 
         # Fill self from config
-        for key, value in data.items():
+        for key, value in list(data.items()):
```

Comment:
```suggestion
        for key, value in data.items():
```
If you cast it to a list, you lose the benefits from the generators

Reply:
Not compliant with python 2.7, Will keep it in our backlog when we will stop supporting 2.7. Thanks for sharing.